onnxscript/values.py

```diff
@@ -258,31 +258,39 @@ class Op(OpLike):
     Attributes:
         opset: The Opset that this op belongs to.
         name: The name of the op.
-        opschema: The ONNX OpSchema for the op.
+        op_schema: The ONNX OpSchema for the op.
     """
 
     def __init__(
-        self, opset: Opset, opname: str, opschema: Optional[onnx.defs.OpSchema] = None
+        self, opset: Opset, opname: str, op_schema: Optional[onnx.defs.OpSchema] = None
     ) -> None:
         self._opset = opset
         self._name = opname
-        self._opschema = opschema
+        if op_schema is None:
+            self._op_schema = opset[opname]
+        else:
+            self._op_schema = op_schema
```

Comment:
Nit: I believe `self._op_schema = op_schema or opset[opname]` does the same thing more compactly.

Reply:
Done